tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -118,6 +117,45 @@ def create_data_catalogue_snapshot(public_url) -> str:
 
     return json.dumps(result, sort_keys=True, indent=2)
 
+def create_all_methodologies_snapshot(public_url) -> str:
+    all_methodologies_url = f"{public_url.rstrip('/')}/methodology"
+    parsed_html = _gets_parsed_html_from_page(all_methodologies_url)
+
+    methodologies_accordion = parsed_html.find(id="publications")
```

Comment:
I wonder of the selector `id="publications"` can be renamed on the frontend to make it more clearer - the other snapshots use `id="themes"`. I assume this is hardcoded in the methodology page frontend code.

Reply:
We can change it if needed. The Accordion with this ID is in `MethodologyIndexPage.tsx` 

```
        <Accordion
          id="publications"
          onSectionOpen={accordionSection => {
            logEvent({
              category: 'Methodologies',
              action: 'Publications accordion opened',
              label: accordionSection.title,
            });
          }}
        >
```

Reply:
I've done this